src/main/java/pl/kedziorek/mpkoperator/MpkOperatorApplication.java

```diff
@@ -10,4 +13,9 @@ public static void main(String[] args) {
         SpringApplication.run(MpkOperatorApplication.class, args);
     }
 
+    @Bean
+    PasswordEncoder passwordEncoder() {
```

Comment:
Wyrzuć to do klasy konfiguracyjnej

Reply:
Stworzona nowa klasa AppConfig i wrzucone tam